README.md

```diff
@@ -5,12 +5,37 @@
 
 recode is a utility-focused fabric mod for [DiamondFire](https://mcdiamondfire.com/home/) to make development more fun. It is the official successor to [CodeUtilities](https://github.com/CodeUtilities/CodeUtilities-2.0), from which it was cloned.
 
-You can find the mod jar [here](https://modrinth.com/mod/recode/versions). Future plans include removing bloat/unnecessary features, improving existing code, adding new features, and migrating to Kotlin.
+Future plans include removing bloat/unnecessary features, improving existing code, adding new features, and migrating to Kotlin.
 
 Feel free to join the recode Discord server: https://discord.gg/GWxWtcwA2C
 
 <!-- modrinth_exclude.start -->
-Or check the Modrinth page: https://modrinth.com/mod/recode
+## Installation
```

Comment:
While putting this in its own area and not having it incorporated into another paragraph is good, we should probably just direct to modrinth as it's more user friendly and I believe it's auto updated when a release is made on GitHub.